Toolset/palettes/script editor/behaviors/revsecommoneditorbehavior.livecodescript

```diff
@@ -1804,13 +1809,23 @@ on optionKeyDown pKey
    keyDown pKey
 end optionKeyDown
 
+local sVScroll
 
 on scrollBarDrag
-   if the short name of the target is "Gutter" or the short name of the owner of the target is "Gutter" then
-      exit scrollBarDrag
-   end if
+   local tVScroll
+   
+   # 2017-JUL-28 bhall2001
+   # update only when vScroll changes. Mouse wheel scrolling sends
+   # vScrolls that don't change. Up arrow causes 2 scrollBarDrag message
+   # so we skip the message when up arrow flag is set.
+   put the vScroll of the target into tVScroll
+   if (tVScroll = sVScroll) or sSkipUpArrow then exit scrollBarDrag
```

Comment:
@bhall2001 the script local variable "sSkipUpArrow" is declared at (new) line 2147 (below this point) and is not seen here and also throws an error when strict compilation is on. It should be declared before first reference to it.